teamscale-client/src/main/java/com/teamscale/client/ITeamscaleService.java

```diff
@@ -105,7 +105,7 @@ Call<ResponseBody> uploadExternalReports(
 	);
 
 	/** Retrieve clustered impacted tests based on the given available tests and baseline timestamp. */
-	@PUT("api/v7.6.0/projects/{projectName}/impacted-tests?prioritization-strategy=CHEAP_ADDITIONAL_COVERAGE_PER_TIME")
+	@PUT("api/v8.0.0/projects/{projectName}/impacted-tests?prioritization-strategy=CHEAP_ADDITIONAL_COVERAGE_PER_TIME")
```

Comment:
As we switched to `CHEAP_ADDITIONAL_COVERAGE_PER_TIME` being the default value i the meantime we can remove the extra parameter here that we only added for backwards compatibility